server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/CassandraDefaultBucketDAO.java

```diff
@@ -19,12 +19,11 @@
 
 package org.apache.james.blob.cassandra;
 
-import static com.datastax.driver.core.ConsistencyLevel.LOCAL_ONE;
-import static com.datastax.driver.core.querybuilder.QueryBuilder.bindMarker;
-import static com.datastax.driver.core.querybuilder.QueryBuilder.delete;
-import static com.datastax.driver.core.querybuilder.QueryBuilder.eq;
-import static com.datastax.driver.core.querybuilder.QueryBuilder.insertInto;
-import static com.datastax.driver.core.querybuilder.QueryBuilder.select;
+import static com.datastax.oss.driver.api.core.ConsistencyLevel.LOCAL_ONE;
```

Comment:
We need to create an optimistic execution profile for this.

Reply:
It's is old code, 
I only changed the import url
`import static com.datastax.driver.core.ConsistencyLevel.LOCAL_ONE;` 
-> 
`import static com.datastax.oss.driver.api.core.ConsistencyLevel.LOCAL_ONE;`